1.15-alpine/Dockerfile

```diff
@@ -1,10 +1,17 @@
 FROM golang:1.15-alpine
 LABEL maintainer="Frank Wilkerson"
 
-RUN apk add --no-cache gcc musl-dev git
+RUN apk add --no-cache gcc musl-dev git bash
 RUN wget -O- -nv https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.38.0
 RUN mv /go/bin/golangci-lint /usr/local/bin/
 RUN wget -O /usr/local/bin/swagger "https://github.com/go-swagger/go-swagger/releases/download/v0.26.1/swagger_linux_amd64"
 RUN chmod +x /usr/local/bin/swagger
 RUN wget -O /usr/local/bin/go-acc "https://github.com/fwilkerson/go-acc/releases/download/v0.2.6/go-acc_linux_amd64"
 RUN chmod +x /usr/local/bin/go-acc
+RUN go get golang.org/dl/gotip
+RUN mv /go/bin/gotip /usr/local/bin/gotip
+RUN chmod +x /usr/local/bin/gotip
+RUN gotip download dev.fuzz
+RUN mkdir /src
+RUN mv /root/sdk /src/sdk
+RUN chmod +x /src
```

Comment:
why is the src directory made executable? Is it because of this line in the fuzz test runner?
```
cmd := exec.Command(fmt.Sprintf("gotip test -fuzz=%s -fuzztime=5s ./tests", funcName))
```